input/aragog.toml

```diff
@@ -68,8 +68,9 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
 
         # radiative equilibrium
         [params.stop.radeqm]
-            enabled = false
-            F_crit  = 0.2     # W m-2, model will terminate when |F_atm| < F_crit
+            enabled = true
```

Comment:
Please add some qualifier here for what is "enabled" or "disabled".

Reply:
Throughout all .toml files

Reply:
Done. Added a qualifier at the start of the `[params.stop]` table.